client/src/app/components/managing-body/managing-body.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="managingBody" (click)="onClick()">
+    <div class="ui-gird">
+        <div class="sixteen wide cloumn">
```

Comment:
@tarieli1 You really can't see if it's right or wrong, we don't have yet 'Semantic UI`. Delete the classes and like I said, we'll open a new issue to work on the style of the front page later on.